gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/KeelService.java

```diff
@@ -86,7 +86,9 @@ Response updateConstraintStatus(
 
   @GET("/application/{application}")
   Map getApplicationDetails(
-      @Path("application") String application, @Query("includeDetails") Boolean includeDetails);
+      @Path("application") String application,
+      @Query("includeDetails") Boolean includeDetails,
```

Comment:
What would happen if the new keel version (I.e. the one that no longer accepts this parameter) got sent `includeDetails=false` on the request?

Reply:
Or actually, I guess the same question applies for a value of `true` - that’s what deck will be sending up until we change it.

Reply:
It just ignores it.